src/components/editions/article.tsx

```diff
@@ -87,12 +87,16 @@ const galleryWrapperStyles = css`
 	}
 `;
 
-const headerBackgroundStyles = (item: Format): SerializedStyles => css`
+const headerBackgroundStyles = (item: Item): SerializedStyles => css`
```

Comment:
Don't think you need this change?

Reply:
If you check the `headerBackgroundStyles` function it expects an `Item` not a `Format` so Typescript was throwing an error here:

```
export const headerBackgroundColour = (item: Item): Colour => {
	const format = getFormat(item);
	[...]
}
```

Reply:
@joecowton1 maybe I'm missing something, but `headerBackgroundColour` doesn't need `Item` either? in fact it needs format, hence uses `getFormat`?

Reply:
Updated 👍